app/assets/javascripts/helpers/fetch-content.js

```diff
@@ -0,0 +1,17 @@
+window.FetchContent = { }
+
+window.FetchContent.govspeak = function (text, path) {
```

Comment:
I was tempted to do so, but the path contains the edition hash, so not very easy to move it into JavaScript

Reply:
> Also, since this is no longer coupled to a component, I think it's reasonable to hard-code the path in here and not have to do the 'data attribute dance'. Still raises the question if the callers are really components anymore, as they rely so much on our app :-|.

I was tempted to hardcode the path, but the path contains the edition hash, so not very easy to move it into JavaScript.

Reply:
Updated as discussed `GovspeakFetch.getBody`